frontend/src/test.ts

```diff
@@ -1,11 +1,11 @@
 // This file is required by karma.conf.js and loads recursively all the .spec and framework files
 
+import 'zone.js/testing'; // This MUST be the first import in the list.
```

Comment:
**question (non-blocking):** Rather than ignoring this whole file from prettier, is there a way to mark this one line as special?

Reply:
This is a great question. I tried to do this [following the documentation](https://prettier.io/docs/en/ignore.html#javascript), and prettier still reformatted it. I wonder if t

> A JavaScript comment of // prettier-ignore will exclude the next node in the abstract syntax tree from formatting.

I wonder if import lines don't become nodes in the AST, and that's why it's still reordering the imports. Happy to think through this more, I don't like that we're ignoring a whole file just for a line.

Reply:
And I wonder if the root cause is with the plugin we're using to sort imports, https://github.com/trivago/prettier-plugin-sort-imports.